kong/plugins/rate-limiting/schema.lua

```diff
@@ -39,9 +39,10 @@ return {
           { limit_by = {
               type = "string",
               default = "consumer",
-              one_of = { "consumer", "credential", "ip", "service", "header" },
+              one_of = { "consumer", "credential", "ip", "service", "header", "path" },
           }, },
           { header_name = typedefs.header_name },
+          { path = typedefs.path },
```

Comment:
The above code does not make use of this config at all and should probably be removed.